options/rtld/generic/linker.cpp

```diff
@@ -1935,4 +2066,3 @@ void Loader::_processLazyRelocations(SharedObject *object) {
 		}
 	}
 }
-
```

Comment:
Removing newline at EOF? `InsertNewlineAtEOF: true` to the rescue?

Reply:
There's two newlines there, and clang-format removes one of them. I think that's fine